src/components/include/utils/threads/message_loop_thread.h

```diff
@@ -86,6 +86,9 @@ class MessageLoopThread {
   // Process already posted messages and stop thread processing. Thread-safe.
   void Shutdown();
 
+  // Added for utils/test/auto_trace_test.cc
+  size_t GetMessageQueueSize() const;
```

Comment:
Let it be a part of public interface.
Using differ interfaces in and out of test is not good practise. 
But in this case we need to add an appropriate unit-test, don't we?